### ISSUE TITLE
Terminology: Define term *bootloader*

### ISSUE THREAD

**Issue by paulmenzel** [UniversalScalableFirmware/documentation#33]
The term *bootloader* is used, but not defined. It seems to be used differently, from what I am used to. (The program originally in the MBR. For example, GRUB is a bootloader.)

**Comment by gdong1** [comment]
Yes, Bootloader could be OS bootloader (Grub, LILO, etc.) for OS folks, or firmware bootloader for firmware developers. The bootloader mentioned in the USF means the boot firmware except payload part.  We could define "Bootloader" as "First program executed after power on, typically stored in read-only memory (ROM)  along with some needed data to initialize random-access memory (RAM), silicon and platform specific configuration. ",  but confusion might still exist since traditional firmware bootloader include payload. 
Do you have any suggestion to have a new name to replace "Bootloader"?
maybe we could use "Pre-Payload firmware" or "Platform Initialization firmware" instead.

**Comment by paulmenzel** [comment]
The [Wikipedia *Bootloader* article](https://en.wikipedia.org/wiki/Bootloader) uses “first-stage bootloaders”. Maybe use that?
